compiler/circle-part-value-test/part_eval_one.py

```diff
@@ -23,6 +24,32 @@
 tflite_model = args.name + ".tflite"
 circle_model = args.name + ".circle"
 partition_conn_ini = args.name + ".conn.ini"
+partition_conn_json = args.name + ".conn.json"
+expected_count = args.name + ".excnt"
+
+# Check expected count of models from partitioning
+try:
+    with open(expected_count, "r") as expected_count_file:
+        expected_count_line = expected_count_file.readline()
+        expected_count_file.close()
```

Comment:
```suggestion
```
This line can be removed because `with` statement does the same thing.

Reply:
Thanks!